src/ResourceManager/Resources/Commands.Resources.Test/ScenarioTests/ActiveDirectoryTests.cs

```diff
@@ -88,7 +87,8 @@ public void TestGetADGroupWithSearchString()
         [Trait(Category.AcceptanceType, Category.CheckIn)]
         public void TestGetADGroupWithBadSearchString()
         {
-            ResourcesController.NewInstance.RunPsTest(interceptor, "Test-GetADGroupWithBadSearchString");
+            //ResourcesController.NewInstance.RunPsTest("Test-GetADGroupWithBadSearchString");
+            TestManager.RunTestScript("Test-GetADGroupWithBadSearchString");
```

Comment:
Because most of the Resource test cases have the same test infrastructure, for clarity of the code, I created a TestManager instance in the base class and inherited it in the test classes.